src/flow/coriolisterm_mod.F90

```diff
@@ -0,0 +1,197 @@
+
+MODULE coriolisterm_mod
+    USE core_mod
+
+    IMPLICIT NONE(type, external)
+    PRIVATE
+
+    ! activity flag and rotation vector "omega"
+    LOGICAL, PROTECTED :: has_coriolis = .FALSE.
+    REAL(realk), PROTECTED :: omega(3)
+
+    ! module functions
+    PUBLIC :: init_coriolisterm, finish_coriolisterm, coriolisterm
+
+CONTAINS
+
+    SUBROUTINE init_coriolisterm()
+
+        IF ( fort7%exists("/flow/coriolis") ) THEN
+
+            ! retrieving rotation rate vector from parameters.json
+            has_coriolis = .TRUE.
+            CALL fort7%get_array("/flow/coriolis/omega", omega)
+
+            ! display obtained parameters
+            IF (myid == 0) THEN
+                WRITE(*, '("CORIOLIS TERM:")')
+                WRITE(*, '(2X, "omega (rotation vector):   ", 3(G0, 1X))') omega
+                WRITE(*, '()')
+            END IF
+        END IF
```

Comment:
I like to use guard clauses whenever possible to avoid having big chunks of code inside IF statements. See for instance init_boussinesqterm. I believe it makes the code simpler.

```suggestion
        has_coriolis = .FALSE.
        IF (.NOT. fort7%exists("/flow/coriolis")) THEN
            RETURN
        END IF
        has_coriolis = .TRUE.

        CALL fort7%get_array("/flow/coriolis/omega", omega)
        IF (myid == 0) THEN
            WRITE(*, '("CORIOLIS TERM:")')
            WRITE(*, '(2X, "omega (rotation vector):   ", 3(G0, 1X))') omega
            WRITE(*, '()')
        END IF
```